virttest/qemu_devices/qdevices.py

```diff
@@ -2064,11 +2065,28 @@ def _dev2addr(self, device):
                     bus = int(busid[1])
         if isinstance(busid, int):
             bus = busid
-        if device.get_param('unit'):
-            unit = int(device.get_param('unit'))
+        if device.get_param(self.unit_spec):
+            unit = int(device.get_param(self.unit_spec))
         return [bus, unit]
 
 
+class QSerialBus(QBusUnitBus):
```

Comment:
Hi @vivianzhu, would you mind explaining the difference between QBusUnitBus and QPCIBus? Why serial bus (do you mean `virtio-serial-pci`?) should be a QBusUnitBus not a QPCIBus? Thanks!